Frameworks/AEPSwiftUI/Sources/Constants.swift

```diff
@@ -0,0 +1,18 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+enum Constants {
+    static let LOG_TAG = "AEPSwiftUI"
+    static let EXTENSION_VERSION = "5.0.0"
```

Comment:
I need to give a little bit more thought about the version.
Should we keep it bump it to 6.0.0 ?

Reply:
all of our ios extensions are 5.x.  i assume you're asking due to the min ios requirement being 12 for swiftui compatibility, but i think it's fine to just keep it 5.x until next spring when they all get updated.